modules/agent-install-networking.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="agent-install-networking_{context}"]
+= About networking
+
+During the initial boot, the machines require an IP address configuration that is set either through a Dynamic Host Configuration Protocol (DHCP) server or statically by
+selecting the below options. If you provide the IP address for the `rendezvousIP` field through the DHCP option, then ensure that it is for the machine's IP that is going to be used for deployment.
```

Comment:
`If you provide the IP address for the `rendezvousIP` field through the DHCP option, then ensure that it is for the machine's IP that is going to be used for deployment.`
Maybe I am thinking too much but the tone of this sentence doesn't look user friendly specifically because of `you` usage.

Reply:
This looks to be according to the guidelines :slightly_smiling_face:

Reply:
As discussed we use _you in docs_ and also follow the IBM style guideline for example >>https://docs.openshift.com/container-platform/4.11/authentication/identity_providers/configuring-request-header-identity-provider.html